src/components/JoinIdeaOverlay/reducer.js

```diff
@@ -17,7 +17,7 @@ export default (state = initialState, action) => {
 			};
 		case actions.close:
 			return {
-				...state,
+				...initialState,
```

Comment:
I don't understand this, what's the difference here between state and initialState?

Reply:
`initialState` is just a variable that I specified in the beginning of the file, and its always the same.  `state` an actual a state at the moment of me trying to change it